genesis/process/shardGenesisBlockCreator.go

```diff
@@ -46,37 +46,69 @@ type deployedScMetrics struct {
-		AheadOfTimeGasUsageEnableEpoch:              unreachableEpoch,
-		BelowSignedThresholdEnableEpoch:             unreachableEpoch,
-		GasPriceModifierEnableEpoch:                 unreachableEpoch,
-		MetaProtectionEnableEpoch:                   unreachableEpoch,
-		TransactionSignedWithTxHashEnableEpoch:      unreachableEpoch,
-		SwitchHysteresisForMinNodesEnableEpoch:      unreachableEpoch,
-		SwitchJailWaitingEnableEpoch:                unreachableEpoch,
-		BlockGasAndFeesReCheckEnableEpoch:           unreachableEpoch,
-		RelayedTransactionsV2EnableEpoch:            unreachableEpoch,
-		BuiltInFunctionOnMetaEnableEpoch:            unreachableEpoch,
-		ComputeRewardCheckpointEnableEpoch:          unreachableEpoch,
-		IncrementSCRNonceInMultiTransferEnableEpoch: unreachableEpoch,
-		SCRSizeInvariantCheckEnableEpoch:            unreachableEpoch,
-		BackwardCompSaveKeyValueEnableEpoch:         unreachableEpoch,
-		RepairCallbackEnableEpoch:                   unreachableEpoch,
-		MetaESDTSetEnableEpoch:                      unreachableEpoch,
-		AddTokensToDelegationEnableEpoch:            unreachableEpoch,
-		MultiESDTTransferFixOnCallBackOnEnableEpoch: unreachableEpoch,
-		SenderInOutTransferEnableEpoch:              unreachableEpoch,
-		OptimizeGasUsedInCrossMiniBlocksEnableEpoch: unreachableEpoch,
-		FixOOGReturnCodeEnableEpoch:                 unreachableEpoch,
-		RemoveNonUpdatedStorageEnableEpoch:          unreachableEpoch,
-		OptimizeNFTStoreEnableEpoch:                 unreachableEpoch,
-		CreateNFTThroughExecByCallerEnableEpoch:     unreachableEpoch,
-		DisableOldTrieStorageEpoch:                  unreachableEpoch,
-		StorageAPICostOptimizationEnableEpoch:       unreachableEpoch,
-		TransformToMultiShardCreateEnableEpoch:      unreachableEpoch,
+		SCDeployEnableEpoch:                    unreachableEpoch,
+		BuiltInFunctionsEnableEpoch:            0,
+		RelayedTransactionsEnableEpoch:         unreachableEpoch,
+		PenalizedTooMuchGasEnableEpoch:         unreachableEpoch,
+		SwitchJailWaitingEnableEpoch:           unreachableEpoch,
+		SwitchHysteresisForMinNodesEnableEpoch: unreachableEpoch,
+		BelowSignedThresholdEnableEpoch:        unreachableEpoch,
+		TransactionSignedWithTxHashEnableEpoch: unreachableEpoch,
+		MetaProtectionEnableEpoch:              unreachableEpoch,
+		AheadOfTimeGasUsageEnableEpoch:         unreachableEpoch,
+		GasPriceModifierEnableEpoch:            unreachableEpoch,
+		RepairCallbackEnableEpoch:              unreachableEpoch,
+		MaxNodesChangeEnableEpoch: []config.MaxNodesChangeConfig{
+			{
+				EpochEnable:            unreachableEpoch,
+				MaxNumNodes:            0,
+				NodesToShufflePerShard: 0,
+			},
+		},
+		BlockGasAndFeesReCheckEnableEpoch:                 unreachableEpoch,
+		StakingV2EnableEpoch:                              unreachableEpoch,
+		StakeEnableEpoch:                                  unreachableEpoch,
+		DoubleKeyProtectionEnableEpoch:                    unreachableEpoch,
```

Comment:
This can be left to 0, there is an additional duplication check system on the read file but having this one activated, will make the genesis block creation fail, thus, having an early warning here

Reply:
done